skin/light/light.css

```diff
@@ -178,6 +175,44 @@
   color: #333 !important;
 }
 
+.tabbrowser-tab.only-fade-in {
+  min-height: 36px;
+  max-height: 36px;
+  animation: 0.5s fade-in;
+  animation-fill-mode: forwards;
```

Comment:
Can we just set `animation-fill-mode: forwards` on the `.tabbrowser-tab {` selector, instead of repeating it in all the sub-selectors?